### PR TITLE
Fixed #4 python 3 support

### LINKED ISSUE #4: Python 3 support
It looks like this repo has a lot of python 2-esque `print foo` statements, which suggests that it's build for python 2.
Why are we still using python 2?

### PR DESCRIPTION
**Pull request by ssbarnea**

### CODE REVIEW

**Comment by ssbarnea** [comment]
I hope you will like the fact this also adds tests for Python 3 on Travis so you will know if future changes will break anything.

File: .travis.yml [hunk 1]
```diff
@@ -1,16 +1,12 @@
 language: python
 python:
  - 2.7
+ - 3.5
 sudo: false
-env:
- - TOX_ENV=py27-unittest
- - TOX_ENV=py27-integration
- - TOX_ENV=lint
```
Comment (ericholscher) [inline .travis.yml]: This should continue to run linking and the other 2.7 tests -- you should add 3.5 tests to this.

File: .travis.yml [hunk 2]
```diff
@@ -1,16 +1,12 @@
...
- #- TOX_ENV=docs
 install:
- - pip install tox
+ - pip install tox-travis
```
Comment (agjohnson) [inline .travis.yml]: Does this not allow local testing? -1 from me on moving this to travis if so. I much prefer local debugging.

File: .travis.yml (hunk 1 above)
Comment (agjohnson) [inline .travis.yml]: `tox-travis` removes the need for this apparently?

**Comment by agjohnson** [comment]
Getting caught up with some widespread debt here. I just rebased and merged this. We're using `tox-travis` for support across python versions on other projects now, so this makes a lot of sense.

Thanks for the addition, sorry it went stale for so long!

**Pull request closed without merging**